README.md

```diff
@@ -25,7 +27,7 @@ git clone git://github.com/TryGhost/Ghost-Vagrant.git
 cd Ghost-Vagrant
 ```
 
-You will now need a copy of Ghost itself:
+You will now need a copy of Ghost itself, cloned into your new local Ghost-Vagrant repo:
 
```

Comment:
How about we also remove the `cd Ghost-Vagrant` from the code block above, and put it in the code block below?